monitor/io.py

```diff
@@ -0,0 +1,43 @@
+import os
+import subprocess
+import numpy as np
+from tempfile import mkstemp
+from shutil import move, rmtree
+
+######### ----------------------------------------###########
```

Comment:
general style comments:

1.  I don't love these comment lines.  I used to use them but have found they just add clutter.  I'd recommend removing all of them in your project.
2.  PEP8.  I'm seeing a number of PEP8 issues (spaces before/after arguments, etc.).  There are programs that will automatically clean up this sort of stuff and I'd suggest you run your python code through one before this PR is completed.